// ng-confirm-box/esm2020/lib/ng-confirm.service.mjs
import { Injectable } from '@angular/core';
import { Subject } from 'rxjs';
import * as i0 from "@angular/core";
export class NgConfirmService {
    constructor() {
        this.displayModa$ = new Subject();
    }
    showConfirm(message, yes, no) {
        this.displayModa$.next({
            isVisible: true,
            messageText: message,
            yes: () => {
                yes();
                this.closeConfirm();
            },
            no: () => {
                no();
                this.closeConfirm();
            },
        });
    }
    closeConfirm() {
        this.displayModa$.next(false);
    }
}
NgConfirmService.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.2.1", ngImport: i0, type: NgConfirmService, deps: [], target: i0.ɵɵFactoryTarget.Injectable });
NgConfirmService.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "14.2.1", ngImport: i0, type: NgConfirmService, providedIn: 'root' });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.2.1", ngImport: i0, type: NgConfirmService, decorators: [{
            type: Injectable,
            args: [{
                    providedIn: 'root'
                }]
        }], ctorParameters: function () { return []; } });
