// ng-confirm-box/esm2020/lib/ng-confirm.module.mjs
import { NgModule } from '@angular/core';
import { NgConfirmComponent } from './ng-confirm.component';
import * as i0 from "@angular/core";
export class NgConfirmModule {
}
NgConfirmModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "14.2.1", ngImport: i0, type: NgConfirmModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
NgConfirmModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "14.0.0", version: "14.2.1", ngImport: i0, type: NgConfirmModule, declarations: [NgConfirmComponent], exports: [NgConfirmComponent] });
NgConfirmModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "14.2.1", ngImport: i0, type: NgConfirmModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "14.2.1", ngImport: i0, type: NgConfirmModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        NgConfirmComponent
                    ],
                    imports: [],
                    exports: [
                        NgConfirmComponent
                    ]
                }]
        }] });
